manager/controllers/default/header.php

```diff
@@ -188,11 +188,11 @@ public function buildMenu($name, $placeholder)
                     $menu['action'] .= '&namespace='.$menu['namespace'];
                 }
                 $onclick = (!empty($menu['handler'])) ? ' onclick="'.str_replace('"','\'',$menu['handler']).'"' : '';
-                $menuTpl .= '<a href="?a='.$menu['action'].$menu['params'].'"'.( $top ? ' class="top-link"': '' ).$onclick.$title.'>'.$label.$description.'</a>'."\n";
+                $menuTpl .= '<a href="?a='.$menu['action'].$menu['params'].'"'.' aria-label="'.$menu['text'].'" '.( $top ? ' class="top-link"': '' ).$onclick.$title.'>'.$label.$description.'</a>'."\n";
```

Comment:
Is $menu['text'] already encoded somewhere to avoid XSS insertion in the menu text?

Reply:
@Mark-H would the following work for @Ibochkarev to address the XSS concern (also adding to lines 193, 195 and 328, below)? 

`' aria-label="'.htmlentities($menu['text'], ENT_QUOTES, 'UTF-8').'" '.`

Reply:
Yeah that's how that would normally be handled. It does look like it's also not being applied to the $title and the $description so this whole thing might be a problem. Or, perhaps it's already handled elsewhere, I haven't gone through the getCache() method reponsible for fetching menus to see if that's sanitising it.